src/Fixer/Operator/BinaryOperatorSpacesFixer.php

```diff
@@ -367,9 +369,9 @@ protected function createConfigurationDefinition(): FixerConfigurationResolverIn
                         if (!\in_array($value, self::$allowedValues, true)) {
                             throw new InvalidOptionsException(
                                 sprintf(
-                                    'Unexpected value for operator "%s", expected any of "%s", got "%s".',
+                                    'Unexpected value for operator "%s", expected any of %s, got "%s".',
                                     $operator,
-                                    implode('", "', self::$allowedValues),
+                                    Utils::naturalLanguageJoin(array_map(HelpCommand::class.'::toString', self::$allowedValues)),
```

Comment:
- I believe that `HelpCommand` shouldn't be called here, it would be better to move `toString()` method to `Utils`
- Then, `toString()` could be used internally within `naturalLanguageJoin()`, or at least `static fn ($value) => Utils::toString($value);` could be used for better usage discoverability (instead of magic callable).

Reply:
See #7034

Reply:
Merged, so please update this PR 🙂.

Reply:
Updated.